test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -44,10 +44,7 @@
 # The min onnx opset version to test for
 MIN_ONNX_OPSET_VERSION = 9
 # The max onnx opset version to test for
-MAX_ONNX_OPSET_VERSION = (
-    _constants.ONNX_MAX_OPSET - 1
-)  # TODO: ORT does not support opset 18 yet
-
+MAX_ONNX_OPSET_VERSION = _constants.ONNX_MAX_OPSET
```

Comment:
onnx ci test doesn't agree lol. The error is on exporter side though. I skimmed at the log and it's mostly some attributes of some reduce ops are promoted to input.

Reply:
Ah, should be the ReduceMean updates on opset 18 ...

Reply:
Blocked by https://github.com/pytorch/pytorch/blob/cbac56e2445e6e9fea906d28889b47595b2611c0/torch/csrc/jit/passes/onnx/shape_type_inference.cpp#L921-L926

The axes becomes input in opset 18 ReduceMean. Do you know how to use API maybe converting at::Tensor to std::vector<int64_t>? Because now I have to update the block to be something like this 

```python
    std::vector<int64_t> axes_vector(rank_0);
    if (opset_version >= 18){
      // axes becomes input since opset 18
      if (n->inputs().at(1)->node()->mustBeNone()) {
        std::iota(axes_vector.begin(), axes_vector.end(), 0);
      } else {
        axes_vector = n->inputs().at(1)->node()->t(attr::value).item().toIntVector(); # INVALID CODE
      }
    } else {
      if (!n->hasAttributeS("axes")) {
        std::iota(axes_vector.begin(), axes_vector.end(), 0);
      } else {
        axes_vector = n->is(attr::axes);
      }
    }
```

Reply:
You can refer to this code

https://github.com/pytorch/pytorch/blob/cbac56e2445e6e9fea906d28889b47595b2611c0/torch/csrc/jit/passes/onnx/shape_type_inference.cpp#L691-L694